graph/pkg/service/v0/groups.go

```diff
@@ -81,6 +82,7 @@ func (g Graph) PostGroup(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	g.publishEvent(events.GroupCreated{GroupID: *grp.Id})
```

Comment:
potential nilpointer dereference?

Reply:
Maybe, I assumed that when no error was returned, that the group must be set but I added a nil check anyways.

Reply:
I agree that `grp` should not be nil when the call does not return an error. But `grp.Id` could be nil in which case `*grp.Id` panics.